development.md

```diff
@@ -11,10 +11,13 @@ go test ./...
 To build the binary, run `make`: this will place a built version of the tool in
 the `bin` directory. If you work for incident.io and have a local instance of
 the app running, then you can point it to your local environment using the
-`--api-key` flag, or an environment variable:
+following environment variable:
 
 ```
+export INCIDENT_API_KEY="inc_development_h3gHe9hegh39h3gh39He9hEgh3gh3gHe9h3gh39h3gh39hE9hegh3gh3Ghe9hEG"
```

Comment:
I'd make this more obviously fake so people realise they have to use their own

Reply:
heghegheghegheghe sounds like some evil demented laugh ❤️ 

but +1

Reply:
done